prompt-note/task/index.js

```diff
@@ -0,0 +1,84 @@
+const prompt = require("prompt-sync")();
+
+let count = 0;
+let listItem = [];
+const resultMessage = () => console.log("종료 되었습니다.");
+
+// 질문
+const question = () => {
+  const message = prompt(
+    "1. 전체메모 조회 / 2. 메모 입력 / 3. 메모 모두 삭제하기 / 4. 메모 수정하기 / 0. 종료 : "
+  );
+  const feedback = +message;
+
+  switch (feedback) {
+    case 1:
+      list();
+      break;
+    case 2:
+      write();
+      break;
+    case 3:
+      remove();
+      break;
+    case 4:
+      update();
+      break;
+  }
+};
+
+// 뒤로가기
+const back = () => {
+  const message = prompt("0. 뒤로가기 / 1: 종료 : ");
+  const feedback = Number(message);
+
+  if (feedback === 0) {
+    question();
```

Comment:
```suggestion
   return question();
```
함수가 끝나야 할 경우에는 return 값 넣는걸 추천!